src/TouchableItem.js

```diff
@@ -18,19 +19,11 @@ type Props = {
   borderless?: boolean,
   pressColor?: string,
   pressOpacity?: number,
-  children?: React.Element<any>,
+  children?: Node,
   style?: Style,
 };
 
-type DefaultProps = {
-  pressColor: string,
-};
-
-export default class TouchableItem extends PureComponent<
-  DefaultProps,
-  Props,
-  void
-> {
+export default class TouchableItem extends PureComponent<Props, void> {
```

Comment:
<Props> is enough

Reply:
What is enough? Based on your other comment, I'm guessing `<Props>` is enough, yes?

Reply:
YEp